tests/test_mesh_creation_functions.py

```diff
@@ -752,6 +827,17 @@ def rotation(t):
     )
     input_file.add(sin_set)
 
+    # extend test case with different meshing strategy
+    create_beam_mesh_curve(
+        input_file,
+        Beam3rHerm2Line3,
+        mat,
+        curve,
+        [1.0, 2.0],
```

Comment:
```suggestion
        [1.0, 2.5],
```
Then we also test that the inveral can have a length other than 1.